lib/modules/dashboard.js

```diff
@@ -88,14 +89,20 @@ module.go = async () => {
 const initialTabID = location.hash.replace('#', '') || 'dashboardContents';
 let selectedTabMenuItem;
 
-export function addTab(tabID, tabName) {
+export function addTab(tabID, tabName, tabModule) {
 	const $tabPage = $('<div class="dashboardPane">').appendTo('#siteTable.linklisting').hide();
 
 	const tabMenuItem = CreateElement.tabMenuItem({
 		text: tabName,
 		className: 'res-dashboard-tab',
 		prepend: true,
 	});
+	// if a module with options has been provided then we automatically append the gear link
+	if (tabModule && tabModule.options) {
```

Comment:
okay. I had understood #2803 as only wanting the settings gear if there were additional options for the module